Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2641,6 +2692,7 @@ def to_context(self):
         else:
             human_readable = None  # type: ignore[assignment]
         raw_response = None  # type: ignore[assignment]
+        indicators_timeline = IndicatorsTimeline().indicators_timeline  # type: ignore[assignment]
```

Comment:
why you need this line? remove it and use directly the `self.indicators_timeline.indicators_timeline`